NBitcoin/TransactionBuilder.cs

```diff
@@ -78,7 +78,7 @@ public DefaultCoinSelector()
 			_Rand = new Random();
 		}
 
-		Random? _Rand;

```

Comment:
should be `Random?`

Reply:
but then why do `Random` works? I guess it means you don't have the new null-detection feature of C# enabled?

Reply:
> I guess it means you don't have the new null-detection feature of C# enabled?

Confirmed; to opt-in for this we should add `<Nullable>enable</Nullable>` to the .csproj file.